src/CmsBundle/Resources/config/doctrine/Content.orm.yml

```diff
@@ -186,6 +179,16 @@ Opifer\CmsBundle\Entity\Content:
             precision: 0
             column: show_in_navigation
     manyToOne:
+        layoutPreview:
```

Comment:
Can you rename this simply preview? Could be used to display preview of content in general as well.